src/sider/historikk/container/HistorikkContainer.tsx

```diff
@@ -75,6 +80,34 @@ const getTextForHistorikk = (
   }
 };
 
+function getDialogmotekandidatHistorikkText(
+  { type, vurdertAv }: DialogmotekandidatHistorikkDTO,
+  person: BrukerinfoDTO
+) {
+  switch (type) {
+    case HistorikkType.KANDIDAT:
+      return `${person.navn} ble kandidat til dialogmøte`;
+    case HistorikkType.UNNTAK:
+      return `${vurdertAv} vurderte unntak fra dialogmøte`;
+    case HistorikkType.IKKE_AKTUELL:
+      return `${vurdertAv} vurderte dialogmøte ikke aktuelt`;
+    case HistorikkType.LUKKET:
+      return `Kandidat til dialogmøte ble lukket av systemet`;
```

Comment:
Her ble jeg litt nysgjerrig på hva dette egentlig betyr 😅 Vet veilederne det? 😊

Reply:
Godt spørsmål. Vi bør kanskje diskutere om vi skal ta med disse. Vi har jo tilsvarende status for dialogmøter når vi lukker disse manuelt, så bør i hvert fall se det i sammenheng. cc @eirikdahlen

Reply:
Endret teksten litt etter samtale med Stine. Bør kanskje bruke "masinelt lukket" som begrep når det gjelder dialogmøter også @eirikdahlen